src/ProjectNeon/Assets/Z_Tests/Features/Effects/StealLifeOnAttackTests.cs

```diff
@@ -18,8 +18,8 @@ public void StealLifeOnAttack_ApplyEffect_LifeIsStolenCorrectly()
         attacker.State.TakeRawDamage(6);
 
         AllEffects.Apply(StealLifeOnAttack(), caster, new MemberAsTarget(attacker));
-        new Attack(5).Apply(attacker, target);
+        new Attack(1).Apply(attacker, target);
```

Comment:
I don't understand these numbers. Why are they randomly changed?

Reply:
@SilasReinagel Reverted them to the previous values. Just forgot to remove a different test case